.github/workflows/release.yml

```diff
@@ -17,9 +17,10 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: '20.10.0'
+          registry-url: 'https://registry.npmjs.org'
       - name: Install dependencies and publish
         env:
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Looks like the key name doesn't match here, i.e. we have the npm token injected into the node token var?

Reply:
Yeah that's expected, see https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry